Formula/aws-google-auth.rb

```diff
@@ -193,11 +193,17 @@ def install
 
     resource("Pillow").stage do
       inreplace "setup.py" do |s|
-        sdkprefix = MacOS.sdk_path_if_needed ? MacOS.sdk_path : ""
         s.gsub! "openjpeg.h",
           "probably_not_a_header_called_this_eh.h"
-        s.gsub! "ZLIB_ROOT = None",
-          "ZLIB_ROOT = ('#{sdkprefix}/usr/lib', '#{sdkprefix}/usr/include')"
+        on_macos do
+          sdkprefix = MacOS.sdk_path_if_needed ? MacOS.sdk_path : ""
```

Comment:
Can't this just be:

```suggestion
          sdkprefix = MacOS.sdk_path_if_needed
```

`MacOS.sdk_path_if_needed` will either return `MacOS.sdk_path` itself or nil.